### PR TITLE
Fix CI/CD

### PR DESCRIPTION
**Pull request by knkski**

### CODE REVIEW

File: charms/pipelines-ui/src/charm.py [hunk 1]
```diff
@@ -93,6 +93,44 @@ def set_pod_spec(self, event):
 
         healthz = f"http://localhost:{config['http-port']}/apis/v1beta1/healthz"
 
+        env = {
```
Comment (DomFleischmann) [inline charms/pipelines-ui/src/charm.py:96]: I feel like some of these values should be juju config options for further configurability, but I might be missing context

File: charms/pipelines-ui/src/charm.py [hunk 2]
```diff
@@ -93,6 +93,44 @@ def set_pod_spec(self, event):
...
+            "ML_PIPELINE_SERVICE_PORT": api["port"],
+            "STREAM_LOGS_FROM_SERVER_API": "false",
+            "VIEWER_TENSORBOARD_POD_TEMPLATE_SPEC_PATH": "/etc/config/viewer-pod-template.json",
+            "VIEWER_TENSORBOARD_TF_IMAGE_NAME": "tensorflow/tensorflow",
+        }
+
+        config_json = json.dumps(
+            {"spec": {"serviceAccountName": "kubeflow-pipelines-viewer"}}
+        )
+
```
Comment (DomFleischmann) [inline charms/pipelines-ui/src/charm.py:133]: Code comments on lines which are not part of the commit in this file:

- L26: `relations` seems like weird name for this if it also includes the basic events
- L44: `send_info` might be a bit vague maybe `send_info_to_pipelines`?
- L59-L75: This should use the [minio_interface library](https://github.com/canonical/minio-operator/blob/master/lib/charms/minio/v0/minio_interface.py) in the minio charm.  See the [argo-controller](https://github.com/canonical/argo-operators/tree/main/operators/argo-controller) charm for an example

**Review by DomFleischmann — COMMENTED** [review]
Also addressed some parts of the code which are not part of this commit

**Comment by knkski** [comment]
Github did not like me attempting to rewrite history including this PR. Moving over to #2

**Pull request closed without merging**